dnf/repo.py

```diff
@@ -551,10 +551,6 @@ def load(self):
         try:
             ret = self._repo.load()
         except RuntimeError as e:
-            if self.skip_if_unavailable:
```

Comment:
Why you change behavior of API function? Now it will raise always.

Reply:
This is not an API change, it always raised before and still does? All I did was remove the log messages, since the error was being logged twice - once here and once when the exception thrown four lines below was processed.

Reply:
Sorry my mistake